EssentialFeed/EssentialFeed/ImageComments/API/ImageCommentsEndpoint.swift

```diff
@@ -0,0 +1,16 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+public enum ImageCommentsEndpoint {
+	case get
+
+	public func url(baseURL: URL, imageID: String) -> URL {
```

Comment:
The API endpoint defines the ID as a UUID. So the API here should reflect this requirement to prevent clients form passing *any* string such as `"this is not a UUID"`:

```suggestion
	public func url(baseURL: URL, imageID: UUID) -> URL {
```